test/test_account_operations.sh

```diff
@@ -9,7 +9,7 @@ echo Create account
 ../bin/near create-account $testaccount
 
 echo Get account state
-RESULT=$(../bin/near state $testaccount -v | ../node_modules/.bin/strip-ansi)
+RESULT=$(yes | ../bin/near state $testaccount -v | ../node_modules/.bin/strip-ansi)
```

Comment:
shouldn't require user interaction, i.e. no `yes |`

Reply:
this is not fixed, why it is resolved?

Reply:
my bad actually, looks like it is fixed here. But tests don't pass on CI because we still ask for consent in commands which are used in scripts.